apps/token-e2e/src/support/staking.functions.js

```diff
@@ -48,32 +49,36 @@ Cypress.Commands.add('staking_validator_page_remove_stake', (stake) => {
     .click();
 });
 
-Cypress.Commands.add(
-  'staking_page_associate_tokens',
-  (amount, approve = false) => {
-    cy.highlight(`Associating ${amount} tokens`);
-    cy.get(ethWalletAssociateButton).first().click();
+Cypress.Commands.add('staking_page_associate_tokens', (amount, options) => {
+  let approve = options && options.approve ? options.approve : false;
+  let type = options && options.type ? options.type : 'wallet';
+
+  cy.highlight(`Associating ${amount} tokens`);
+  cy.get(ethWalletAssociateButton).first().click();
+  if (type === 'wallet') {
     cy.get(associateWalletRadioButton, { timeout: 30000 }).click();
-    cy.get(tokenAmountInputBox, epochTimeout).type(amount);
-    if (approve) {
-      cy.get(tokenInputApprove, txTimeout).should('be.enabled').click();
-      cy.contains('Approve $VEGA Tokens for staking on Vega').should(
-        'be.visible'
-      );
-      cy.contains(
-        'Approve $VEGA Tokens for staking on Vega',
-        epochTimeout
-      ).should('not.exist');
-    }
-    cy.get(tokenSubmitButton, epochTimeout).should('be.enabled').click();
-    cy.contains(
-      'can now participate in governance and nominate a validator',
-      txTimeout
-    ).should('be.visible');
+  } else if (type === 'contract') {
+    cy.get(associateContractRadioButton, { timeout: 30000 }).click();
+  } else cy.highlight(`${type} is not association option`);
```

Comment:
nitpick: inconsistent usage of braces and bracket here. Can we make this consistent please